.github/workflows/cs_check.yml

```diff
@@ -0,0 +1,31 @@
+name: "Coding Style Check"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'app'
+      - 'system'
+
+jobs:
+  build:
+    name: PHP ${{ matrix.php-versions }}
```

Comment:
Since you have just specified 7.4 we don't need to use a matrix here. Use something meaningful for the name, like "Run Check"

Reply:
fixed.